scripts/make_new_release.sh

```diff
@@ -7,12 +7,8 @@ set -euo pipefail
 # Enable recursive globbing, and make globs that do not match return null values.
 shopt -s globstar nullglob
 
-# Make sure the current working directory is the root directory.
-if [ ! -f "./requirements.txt" ] || [ ! -f "./CHANGELOG.md" ]; then
-    echo -e 'Please run this script from the root directory of the repo:\n'
-    echo -e '    ./scripts/make_new_release.sh\n'
-    exit 1
-fi
+# Make sure the current working directory for this script is the root directory.
+cd "$(git rev-parse --show-toplevel)"
```

Comment:
Maybe consider putting `cd $(git -C $(dirname $(realpath "${0}") ) rev-parse --show-toplevel )`

Reply:
Fun fact: `realpath` does not exist by default on OSX. Going to remove that bit for now since the workaround is more work than it's worth at the moment.